include/mbedtls/config.h

```diff
@@ -2254,6 +2254,20 @@
  */
 #define MBEDTLS_ECP_C
 
+/**
+ * \def MBEDTLS_USE_UECC
+ *
+ * Enable the tinycrypt ECC library.
```

Comment:
Can we please give the source/origin of where we took the source, so others can maintain it independently? Specifically add a link for https://github.com/intel/tinycrypt

Reply:
We also need to state it's licensed under BSD 3 clause, in case it matters to 3rd parties.

Reply:
See https://github.com/ARMmbed/mbedtls-restricted/pull/565